### PR TITLE
Add companies starting from alphabet "a"

### PR DESCRIPTION
**Pull request by Saphall**
## What kind of change does this PR introduce? 

- [ ] Bug Fix
- [ ] Documentation change
- [x] Added new company
- [ ] Other(s)

## Why was this change needed?
Added some tech companies starting from the alphabet: `a`.

### CODE REVIEW

File: data/companies.json [hunk 1]
```diff
@@ -32,6 +64,70 @@
...
+        "github": null,
+        "instagram": null,
+        "linkedin": "https://www.linkedin.com/company/amnil-technologies-pvt-ltd-/",
+        "twitter": "https://twitter.com/amniltech",
+        "youtube": null
+      }
+    },
+    {
+      "name": "Amsoft Technologies",
+      "establishedAt": null,
```
Comment (thenishantsapkota) [inline data/companies.json:85]: Please research and provide an `establishedAt` for this company

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

File: data/companies.json [hunk 2]
```diff
@@ -32,6 +64,54 @@
...
+        "youtube": null
+      }
+    },
+    {
+      "name": "Awecode",
+      "establishedAt": 1388513700,
+      "location": "Mahadevsthan, Koteshwar, Kathmandu, Nepal",
+      "website": "https://awecode.com/",
+      "email": "hi@awecode.com",
+      "phone": null,
```
Comment (thenishantsapkota) [inline data/companies.json:105]: Also we need the phone number here.

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

File: data/companies.json (hunk 2 above)
Comment (Saphall) [reply data/companies.json:105]: No Phone no. info info on their site. However, adding the number according to info provided in their vacancy post: https://www.linkedin.com/posts/awecode_join-our-team-now-hiring-flutter-intern-activity-7084492296359936002-nk80?utm_source=share&utm_medium=member_desktop.

Comment (Saphall) [reply data/companies.json]: Also @thenishantsapkota , for some companies with more than 1 phone numbers, the `phone` key should be `list` type !

**Review by thenishantsapkota — APPROVED** [review]
LGTM!

**Pull request merged**